src/main.dart

```diff
@@ -1,8 +1,99 @@
-// TODO: Run task list app
-void main(List<String> args) {
-  print('Hello World');
+import 'dart:developer';
+
+import 'dart:io';
+
+class Task {
+  int? taskID;
+  String? name;
+  String? description;
+  bool? isCompleted;
+
+  Task(this.taskID, this.name, this.description, this.isCompleted);
```

Comment:
Use default constructor
```suggestion
  Task({
        this.taskID = 0,
        required this.name,
        required this.description,
        this.isCompleted = false
    });
```